src/Microsoft.VisualBasic/tests/VB/DriveInfo.Unix.Tests.vb

```diff
@@ -0,0 +1,30 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Imports Microsoft.VisualBasic.FileIO
+Imports System
+Imports System.Linq
+Imports Xunit
+
+Namespace Microsoft.VisualBasic.Tests
+
+    Public NotInheritable Class DriveInfoUnixTests
+
+        <Fact>
+        <PlatformSpecific(TestPlatforms.AnyUnix)>
+        Public Shared Sub TestGetDrives()
+            Dim DriveList() As IO.DriveInfo = FileSystem.Drives().ToArray
+            Assert.NotNull(DriveList)
+            Assert.[True](DriveList.Length > 0, "Expected at least one drive")
```

Comment:
why are there braces around true ?

Reply:
The C# to VB translator put them in. Removed.